src/app/layout.tsx

```diff
@@ -22,7 +23,8 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={`font-sans ${inter.variable}`}>
+      <body className={`font-sans ${outfit.variable}`}>
```

Comment:
I'm going to install clsx in the repository, so this will be dated soon.

Reply:
for the time being, we can use this